tests/integration/test_github.py

```diff
@@ -282,6 +287,19 @@ def test_create_release(self):
         assert release.body == "testing release"
         assert count_before + 1 == count_after
 
+    def test_edit_release(self):
+        release = self.hello_world_project.get_release(tag_name="0.1.0")
+        origin_name = release.title
+        origin_message = release.body
+
+        release.edit_release(name="changed name", message="edited message")
```

Comment:
```suggestion
        release.edit_release(name=f"{origin_name}-changed", message=f"{origin_message}-changed")
```

Just to be sure we change anything...